scss/atoms/buttons/_buttons.scss

```diff
@@ -0,0 +1,102 @@
+/*
+* ==========================================================================
+* Buttons
+* ==========================================================================
+*/
+@import "mixins";
+
+/* These colors won't remain here, will be removed with jsass  */
+$background: #fff;
+$chassis-blue: #4fc0c8;
+$chassis-gray-light: #f2f2f2;
+$chassis-gray-dark: #383838;
+
```

Comment:
I'm wondering if we should name the variables based on their purpose rather than color? This way when they get changed in the theme roller a person wouldn't create a purple button that is called "blue".